src/main/cli/settings.c

```diff
@@ -183,7 +184,8 @@ const char * const lookupTableGyroHardware[] = {
     "BMI270",
     "LSM6DSO",
     "LSM6DSV16X",
-    "VIRTUAL"
+    "VIRTUAL",
+    "IIM42653"
```

Comment:
From history, it seems that VIRTUAL is always placed lasts. VIRTUAL is not used on any hardware, so it is not causing config problems (It is IMO only tradition. Maybe it's time to break it)
```suggestion
    "IIM42653",
    "VIRTUAL"
```

Version if VIRTUAL is not kept last (extra comma makes future diffs cleaner):
```suggestion
    "VIRTUAL",
    "IIM42653",
```